resources/assets/coffee/osu_common.coffee

```diff
@@ -158,9 +158,14 @@
 
   timeago: (time) ->
     el = document.createElement('time')
-    el.classList.add 'timeago'
-    el.setAttribute 'datetime', time
-    el.textContent = time
+
+    if moment(0).isSame(time)
```

Comment:
then check if null instead. Not sure if should be here or before this function is called though.

(also wondering why not just `time.getTime() == 0` / `$date->getTimestamp() === 0`)